charts/qdrant/values.yaml

```diff
@@ -27,11 +27,13 @@ service:
       targetPort: 6333
       protocol: TCP
       checksEnabled: true
+      # appProtocol: http
```

Comment:
should not we have the default value?

btw, where `http2` values is coming from? I have looked through https://www.iana.org/assignments/service-names-port-numbers/service-names-port-numbers.txt but I could not spot it.

Reply:
Me neither, seems to be a gke thing: https://github.com/qdrant/qdrant-helm/issues/304

Reply:
yeah, https://github.com/GoogleCloudPlatform/professional-services/blob/f935ef520820895ec6777dd66eaa9b3ec6f1e423/examples/gclb-callouts/manifests/service-extension.yaml#L82